### PR TITLE
feat!: remove importer from default apps

### PR DESCRIPTION
**Pull request by JammingBen**
BREAKING CHANGE: removes `importer` from the default web apps. The app is now available via the [web-extensions repository](https://github.com/owncloud/web-extensions). See https://github.com/owncloud/web-extensions/pull/61.

refs https://github.com/owncloud/web/issues/11698

### CODE REVIEW

File: dev/docker/ocis.web.config.json [hunk 1]
```diff
@@ -28,18 +28,7 @@
...
-          "image/x-ms-bmp"
-        ]
-      }
-    },
-    {
-      "id": "importer",
-      "path": "web-app-importer",
-      "config": {
-        "companionUrl": "https://host.docker.internal:9200/companion"
+        "mimeTypes": ["image/tiff", "image/bmp", "image/x-ms-bmp"]
```
Comment (kulmann) [inline dev/docker/ocis.web.config.json:31]: Not your code, but we have these three mime types hardcoded for thumbnails and for the preview app by now (see e.g. the `mimeTypes.ts` file in the preview app). So we can remove the `preview` app from the external apps section again and just add `preview` to the string-array under the `apps` key. Could you do that here? Only dev stack, so it doesn't need changelog anyway.

**Review by kulmann — COMMENTED** [review]
Can't we go back to upstream `uppy` now with companion being removed from the web repo? Or is it still needed in the runtime for the `importer` app to work correctly?

**Comment by JammingBen** [comment]
> Can't we go back to upstream `uppy` now with companion being removed from the web repo? Or is it still needed in the runtime for the `importer` app to work correctly?

It's still needed in the runtime to work unfortunately :(

**Review by kulmann — APPROVED** [review]
Thanks for taking care 🥇

**Pull request merged**